### PR TITLE
Add acme certificate module

### PR DESCRIPTION
**Pull request by Dusty-Meg**
Add module to create certificates, works with Lets Encrypt

### CODE REVIEW

**Review by JoeCSykes — APPROVED** [review]

File: modules/aws/acme_certificate/variables.tf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  type        = string
+}
+
+variable "aws_hosted_zone_id" {
+  description = "Hosted zone Id of the domain"
+  type        = string
+}
+
+variable "base_domain_name" {
+  description = "Domain without www. or https to create a certificate for"
```
Comment (robg-test) [inline modules/aws/acme_certificate/variables.tf:12]: Is this description correct?

Comment (Dusty-Meg) [reply modules/aws/acme_certificate/variables.tf]: Rejigged the description and added validation to make it easier to read and use.

File: modules/aws/acme_certificate/variables.tf [hunk 2]
```diff
@@ -0,0 +1,30 @@
+variable "email_address" {
```
[the remark this answers is not in the document]
Comment (Dusty-Meg) [reply modules/aws/acme_certificate/variables.tf:1]: Added

**Review by chrisbloe — APPROVED** [review]
Thanks for the validation :).

**Pull request merged**